ethereum_history_api/circuits/lib/src/misc/U256.nr

```diff
@@ -0,0 +1,43 @@
+use crate::misc::types::U256;
+use crate::misc::arrays::pad_left;
+use dep::std::wrapping_add;
+use dep::u2b::u64_to_u8;
+
+global MAX_U8: u8 = 255;
+
+pub(crate) fn is_overflow(left: u8, right: u8) -> bool {
```

Comment:
`is_overflow`, `overflowing_add`, `overflowing_add` is not related to `U256` and works on `u8` type. Maybe some u8 util file...